cpp/cmake/thirdparty/get_raft.cmake

```diff
@@ -92,8 +92,8 @@ endfunction()
 # To use a different RAFT locally, set the CMake variable
 # CPM_raft_SOURCE=/path/to/local/raft
 find_and_configure_raft(VERSION          ${CUML_MIN_VERSION_raft}
-                        FORK             rapidsai
-                        PINNED_TAG       branch-${CUML_BRANCH_VERSION_raft}
+                        FORK             cjnolet
+                        PINNED_TAG       imp-2302-raft_device_resources
```

Comment:
@dantegd, @cjnolet, should we merge the PR with this change and revert it in a follow-up?

Or should we remove it first?

FWIW, this PR is holding up a few things, so if we can revert it in a follow-up, I'd vote for that.

Reply:
I think we can revert it in a follow up, but will wait on @cjnolet input

Reply:
I can open another PR in a follow-up, that way we can unblock other things in the meantime.